### PR TITLE
- added open var scrollsToBottomOnFirstLayout

### PR DESCRIPTION
**Pull request by FraDeliro**
Added open var scrollsToBottomOnFirstLayout, if true messagesCollectionView scrollToBottom.

Thank you for your replay, it's one my first contributions, I have made the changes you have requested.
Let me know if everything is ok.
Thanks ;)
#183

### CODE REVIEW

File: Sources/MessagesViewController.swift [hunk 1]
```diff
@@ -82,6 +84,13 @@ open class MessagesViewController: UIViewController {
             addKeyboardObservers()
             messagesCollectionView.contentInset.bottom = messageInputBar.frame.height
             isFirstLayout = false
+            
+            //scroll to bottom at first load
+            guard scrollsToBottomOnFirstLayout == true else {
```
Comment (SD10) [inline Sources/MessagesViewController.swift]: Can we just make this read as:
```Swift
if scrollsToBottomOnFirstLayout {
    messagesCollectionView.scrollsToBottom(animated: false)
}
```

I'd also like explicitly set `isFirstLayout` at the very end of this scope

File: Sources/MessagesViewController.swift [hunk 2]
```diff
@@ -82,6 +84,13 @@ open class MessagesViewController: UIViewController {
...
+                return
+            }
+            scrollsToBottomOnFirstLayout = false
```
Comment (SD10) [inline Sources/MessagesViewController.swift]: let's not modify the bool after we scroll to bottom, all we're really concerned with is the `isFirstLayout` flag

**Review by SD10 — CHANGES_REQUESTED** [review]
@FraDeliro Thanks for taking care of this 💯 Just a few nit revisions. Can you also add a CHANGELOG entry matching the same style we are using in the `v0.9.0` branch CHANGELOG? This would go under **added** and is not a breaking change

**Review by SD10 — APPROVED** [review]
@FraDeliro Thank you for taking care of this for us 👍

**Comment by FraDeliro** [comment]
@SD10 Thanks for your library ;)

**Comment by fishcharlie** [comment]
What is the status of this? This code doesn't look to be in the library currently.

**Pull request merged**